dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/cslb/CslbTransformationPipelineBuilder.scala

```diff
@@ -41,19 +41,21 @@ object CslbTransformationPipelineBuilder extends PipelineBuilder[Args] {
         s"${args.inputPrefix}/records/*.json"
       )
 
-    // Group by study ID (record number) and field name
-    // to get the format: (studyId, Iterable((fieldName, Iterable(value))))
+    // Group by study ID (record number) and arm (address_year_month)
+    // to get the format: (studyId, arm_id, Iterable((fieldName, Iterable(value))))
     rawRecords
-      .groupBy(_.read[String]("record"))
+      .groupBy(record => {
```

Comment:
⛏️ Given that this is a dupe of the code in env, we should ticket up DRY-ing this.

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-1817 - ticket in Sprint 69 to address this